.gitignore

```
@@ -64,6 +64,20 @@ target/
 .venv*
 venv*
 
+# .env
+.env
+
 # further build artifacts
 lockfiles/
 
+bin/
+dbd/
+db/
+lib/
+*O.linux-x86_64*
+*O.Common*
+sim/areaDetectorIOC/iocBoot/iocexUser1/envPaths
```

Other IOCs also generate `envPaths` files at build time.

```suggestion
*envPaths
```